demosplan/DemosPlanCoreBundle/Logic/Import/Statement/AbstractStatementFromRowBuilder.php

```diff
@@ -71,4 +72,8 @@ abstract public function setNumberOfAnonymVotes(Cell $cell): ?ConstraintViolatio
      */
     abstract public function buildStatementAndReset(): StatementInterface|ConstraintViolationListInterface;
 
+    public function reset(): void
+    {
+        $this->statement = new Statement();
+    }
```

Comment:
I proposed making this non-abstract, but now it think it's wrong. One child class needs to implement this and the other one needs to call `$this->baseStatementFromRowBuilder->reset()`, or am I wrong?

Reply:
done in [e4c767a](https://github.com/demos-europe/demosplan-core/pull/2575/commits/e4c767ad2a943ab771c3fd9a37ca27accb65955e)

Reply:
done in 8ae3fff923d885cba0c253fa0f60141c08909b34